amundsen_application/api/v0.py

```diff
@@ -4,18 +4,30 @@
 from flask import current_app as app
 from flask.blueprints import Blueprint
 
+from amundsen_application.api.metadata.v0 import USER_ENDPOINT
+from amundsen_application.api.utils.request_utils import request_wrapper
 from amundsen_application.models.user import load_user
 
+REQUEST_SESSION_TIMEOUT = 10
```

Comment:
How about adding unit here? `REQUEST_SESSION_TIMEOUT_SEC`

Also, 10 sec on this call seems very high considering p95 latency is under 300ms. Are we using 10 sec for most of metadata service call?

Reply:
I can reduce this to something a bit more reasonable like 3s, but I think the 10s value originates from data preview which can take a long time. It makes sense to configure these separately.

Reply:
Yeah, preview is an outlier and I believe we configured it as 2 minutes.